cost-analyzer/templates/NOTES.txt

```diff
@@ -1,9 +1,35 @@
 
 
 --------------------------------------------------
+{{- $node := (lookup "v1" "Node" "" "") }}
+{{- $isEKS := (regexMatch ".*eks.*" (.Capabilities.KubeVersion | quote) )}}
+{{- $isGT22 := (semverCompare ">=1.23-0" .Capabilities.KubeVersion.GitVersion) }}
+{{- $PVNotExists := (empty (lookup "v1" "PersistentVolume" "" "")) }}
+{{- $EBSCSINotExists := (empty (lookup "apps/v1" "Deployment" "kube-system" "ebs-csi-controller")) }}
+
 {{- $servicePort := .Values.service.port | default 9090 -}}
 Kubecost has been successfully installed.
 
+{{ if (and $isEKS $isGT22) -}}
+
+WARNING: ON EKS v1.23+ INSTALLATION OF EBS-CSI DRIVER IS REQUIRED TO MANAGE PERSISTENT VOLUMES. LEARN MORE HERE: https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html
+
+{{ if (and $EBSCSINotExists $PVNotExists) -}}
+
+ERROR: MISSING EBS-CSI DRIVER WHICH IS REQUIRED ON EKS v1.23+ TO MANAGE PERSISTENT VOLUMES. LEARN MORE HERE: https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html
+
+{{ else if (and $EBSCSINotExists (not $PVNotExists)) -}}
+
+ERROR: MISSING EBS-CSI DRIVER WHICH IS REQUIRED ON EKS v1.23+ TO MANAGE PERSISTENT VOLUMES. LEARN MORE HERE: https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html
+
+{{ else if (not $EBSCSINotExists) -}}
+
+We detected a EBS-CSI Driver, but you should check that Kubecost's PersistentVolume has been installed correctly
```

Comment:
What's the value in this message? Is it needed?

Reply:
I don't know. I would want an opinion on whether the ebs CSI driver 's name is often changed or if it is installed in other namespaces sometimes.

Reply:
In my test on EKS 1.23 and EBS CSI is installed. This message still appears. I think it is a little bit confusing to users/customers. In most cases the EBS CSI driver's name should not be changed or it is deployed outside of kube-system as it is not AWS recommendation. If it is the case, even AWS supports team will recommend users to follow AWS instructions.

Reply:
OK, I can remove! Thanks for the insight.